tests/integration.rs

```diff
@@ -262,6 +271,24 @@ fn looping() -> Result<()> {
     .run()
 }
 
+#[test]
+fn open() -> Result<()> {
+  Test::new()?
+    .program("resize:4:4 save:test.png open:test.png")
+    .env_var("DEGENERATE_OPEN_COMMAND", "echo")
+    .expected_stdout("test.png\n\n")
```

Comment:
This is a little weird. Are the two `\n`s required because it gets passed to unindent?

```suggestion
    .expected_stdout("test.png\n\n")
```

Reply:
Yep that's why. Could fix this by also unindenting the stdout but it breaks other tests